source/write-data-to-mongo.txt

```diff
@@ -0,0 +1,172 @@
+.. _java-rs-write:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the {+driver-short+} to write data to MongoDB.
+   :keywords: usage examples, save, crud, create, code example
+
+.. TODO: change toctree links when other pages are merged in.
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+
+..    /write/insert
+..    /write/update
+..    /write/replace
+..    /write/delete
+..    /write/bulk-write
+..    /write/gridfs
+
+Overview
+--------
+
+This page contains copyable code examples of {+driver-short+} methods that you can use to
+write data to MongoDB.
+
+.. tip::
+
+  To learn more about any of the methods shown on this page, see the link
+  provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <java-rs-write-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<connection string>``, with
+the relevant values for your MongoDB deployment.
+
+This guide uses the Project Reactor library to consume ``Publishers`` returned
+by the {+driver-short+} methods. To learn more about the Project Reactor library
+and how to use it, see `Getting Started <https://projectreactor.io/docs/core/release/reference/#getting-started>`__
+in the Reactor documentation. 
+
+There are also other ways to consume ``Publishers``. You can use one of many alternative libraries such as
+RxJava <https://github.com/ReactiveX/RxJava>__ to consume ``Publishers``, or consume ``Publishers``
```

Comment:
This hyperlink appears broken:
<img width="479" alt="Screenshot 2024-07-18 at 10 16 03" src="https://github.com/user-attachments/assets/bc3372a9-40c8-4a28-bbd5-d2849a0a2161">